adaptor-rm-webapp/pom.xml

```diff
@@ -88,11 +90,6 @@
 			<artifactId>jersey-container-servlet</artifactId>
 			<version>${version.jersey}</version>
 		</dependency>
-		<dependency>
```

Comment:
wait, why did we remove the HK2 DI dep?

Reply:
The earliest version of jersey-hk2 is 2.26. Code runs fine without it if we use 2.25.
But now I realize we have a bigger problem. We will need CQs if we go for Jersey 2.25, since Jersey was migrated to Eclipse in 2.27.
And that also made me think that we did NOT file a CQ for Swagger.

Reply:
so do I have your blessing on this patch?

Reply:
yes, please go ahead